turbo.json

```diff
@@ -5,7 +5,8 @@
 			"cache": false
 		},
 		"build": {
-			"dependsOn": ["^build"]
+			"dependsOn": ["^build"],
+			"outputs": ["dist/**"]
```

Comment:
why do we need this option?

Reply:
I found it was caching packages when it shouldn't and this fixed it. In my case: `packages/workflow` was being cached, when it actually needed to be rebuilt. `packages/core build` was failing because it depended on changes in `packages/workflow`.